### PR TITLE
Update new-teams-desktop-admin.md

### PR DESCRIPTION
**Pull request by baardhermansen**
- Removed duplicate issue
- Replaced all smart quotes
- Added PowerShell coding brackets where that was missing
- Removed a misleading PowerShell code
- General tyding up (removal of extra line breaks, removal of trailing spaces)

### CODE REVIEW

File: Teams/new-teams-desktop-admin.md [hunk 1]
```diff
@@ -239,84 +222,78 @@ There are a few policies that could alter these registry keys and block app inst
...
-- If you have an MSIX client. 
-
+- If you have an MSIX client.
 
 ##### How do I know which one of the above policies is blocking me?
 
-1. Open logs in this path: %appdata%/Microsoft/Teams 
-1. Open logs.txt 
+1. Open logs in this path: `%appdata%/Microsoft/Teams`
+1. Open *logs.txt*
```
Comment (JohanFreelancer9) [inline Teams/new-teams-desktop-admin.md:254]:
```suggestion
1. Open logs in this path: `%appdata%/Microsoft/Teams`.
1. Open *logs.txt*.
```

File: Teams/new-teams-desktop-admin.md [hunk 2]
```diff
@@ -239,84 +222,78 @@ There are a few policies that could alter these registry keys and block app inst
...
-Issue: Windows 10 users may receive the error “We’ve run into an issue” when they download and install the new Teams.</br>
-Action: [Download and install WebView2 Runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section). Then restart the Teams desktop app and try again.
+**Issue**: Windows 10 users may receive the error "We've run into an issue" when they download and install the new Teams.  
+**Action**: [Download and install WebView2 Runtime](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section). Then restart the Teams desktop app and try again.
 
 #### Some people don't see the toggle to opt in
 
 Make sure the user has the minimum requirements met on their computer.  Next, have them sign out and back into the Teams desktop app.  
-If the toggle still doesn't appear, then 
+If the toggle still doesn't appear, then
```
Comment (JohanFreelancer9) [inline Teams/new-teams-desktop-admin.md:285]:
```suggestion
If the toggle still doesn't appear, then:
```

File: Teams/new-teams-desktop-admin.md [hunk 3]
```diff
@@ -329,21 +306,24 @@ As we improve the client, the experience has been improved to align with similar
...
 
 ### General
 
-**Issue**: Presence and Chat/Call from a user's live persona card (LPC) in Outlook doesn't work when non-admin users install the new Teams client. These experiences are also broken while switching between new Teams and classic Teams.</br>
-**Fix**: Confirm that the minimum prerequisites have been met, including installing the Office and Windows security updates as it applies to your organization.</br>See [**Prerequisites**](#prerequisites) section in this article. 
+**Issue**: Presence and Chat/Call from a user's live persona card (LPC) in Outlook doesn't work when non-admin users install the new Teams client. These experiences are also broken while switching between new Teams and classic Teams.
+
+**Fix**: Confirm that the minimum prerequisites have been met, including installing the Office and Windows security updates as it applies to your organization.
+
+See [**Prerequisites**](#prerequisites) section in this article.
```
Comment (JohanFreelancer9) [inline Teams/new-teams-desktop-admin.md:326]:
```suggestion
See the [**Prerequisites**](#prerequisites) section in this article.
```

File: Teams/new-teams-desktop-admin.md [hunk 4]
```diff
@@ -354,34 +334,35 @@ If you discover any accessibility gaps, select **Give Feedback**.
 
 - **Issue**: If custom apps are used, their icons are broken on the left pane.
 
-- **Issue**: You can't install or uninstall any app in new Teams yet. Only apps installed in classic Teams will show up in new Teams. </br>Workaround: Add your app in classic Teams. It will then appear in new Teams. 
+- **Issue**: You can't install or uninstall any app in new Teams yet. Only apps installed in classic Teams will show up in new Teams.</br>
+  **Workaround**: Add your app in classic Teams. It will then appear in new Teams.
 
 - **Issue**: While all the basic capabilities within the app bar and flyout are supported, other advanced capabilities such as pinning, re-ordering, uninstalling, store navigation are still pending.
```
Comment (JohanFreelancer9) [inline Teams/new-teams-desktop-admin.md:340]:
```suggestion
- **Issue**: While all the basic capabilities within the app bar and flyout are supported, other advanced capabilities such as pinning, re-ordering, uninstalling, and store navigation are still pending.
```

File: Teams/new-teams-desktop-admin.md [hunk 5]
```diff
@@ -482,17 +458,17 @@ Details: The meeting attendees may see preview content in those notifications.</
 - **Issue**: Limited options on Team Channel properties Dialog, including:
   - Pin, Manage Channel and Get Email Address available
```
Comment (JohanFreelancer9) [inline Teams/new-teams-desktop-admin.md:459]:
```suggestion
  - Pin, Manage Channel, and Get Email Address available
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @baardhermansen, kindly commit the few minor additional changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@JoanneHendrickson
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by baardhermansen** [comment]
Closing this, there are too many differences since i first submitted the PR.

**Pull request closed without merging**